card/mod/05_standard/spec/set/right/stats_spec.rb

```diff
@@ -5,7 +5,7 @@
   context "core view" do
     it 'should have a table' do
       Card::Auth.as_bot do
-        @core = render_card :core, :name=>"A+*stats"
+        @core = render_card :core, name: "A+*stats"
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.